frontend/src/components/home/community/Community.js

```diff
@@ -0,0 +1,70 @@
+import React, { Component } from "react";
+import { Grid } from "@material-ui/core";
+
+import community from "../../../assets/icons/1.png";
+import networking from "../../../assets/icons/2.png";
+import education from "../../../assets/icons/3.png";
+import future from "../../../assets/icons/4.png";
+
+import "./community.scss";
+
+export default class Community extends Component {
+  render() {
+    return (
+      <div className={"community"}>
+        <Grid
+          container
+          className={"container-community community-text"}
+          spacing={6}
+        >
+          <Grid item xs={12} sm={6} md={3} lg={3} className={"box-community"}>
+            <div className={"card-community"}>
+              <img src={community} alt="Community" />
+              <p className={"title"}>Community</p>
+
+              <p className={"text"}>
+                We encourage developers, technologists, cypher punks, coders,
+                crypto-economists, disegners, makers, and the like to apply. Our
+                ecosystem is bottom up. Emergent.
+              </p>
+            </div>
+          </Grid>
+          <Grid item xs={12} sm={6} md={3} lg={3} className={"box-community"}>
```

Comment:
```suggestion
<Grid item xs={12} sm={6} md={3} lg={3} className={"box-community"}>
            <div className={"card-community"}>
              <img src={community} alt="Community" />
              <p className={"title"}>Community</p>

              <p className={"text"}>
                We encourage developers, technologists, cypher punks, coders,
                crypto-economists, disegners, makers, and the like to apply. Our
                ecosystem is bottom up. Emergent.
              </p>
            </div>
          </Grid>
```
Se puede crear un componente stateless y loopear en lugar de escribir todo el componente multiples veces